README.md

```diff
@@ -23,11 +23,13 @@ jobs:
     name: calibreapp/image-actions
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-      - name: calibreapp/image-actions
-        uses: docker://calibreapp/github-image-actions
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Checkout Repo
+        uses: actions/checkout@master
+
+      - name: Compress Images
+        uses: calibreapp/image-actions@master

```

Comment:
We **must** use the docker built image - GitHub does not cache the docker build so it would mean that each image-actions run would have to wait for it. Because we're compiling from source, that's a 10 minute wait.

Reply:
Turns out this is handled by `actions.yml`. 

What will happen to existing actions using the `docker://` URL?

Reply:
this is taken care of by the `action.yml`